### ISSUE TITLE
About the accuracy

### ISSUE THREAD

**Issue by priceee** [charusharma1991/SSL_PointClouds#9]
Thanks for your excellent work. I noticed that the accuracy in the paper is written in a range (like  81.50±2.25%). I wonder when you get these accuracy, did you use random classes in every time you run the experiments, or you just fix the classes?  And how many times you repeated ?

**Comment by priceee** [comment]
It seems that there is no random choosing of classes in the published code? Changing the "train_K_way" and "test_K_way" in the parser only changes the output shape of the network, but the data loaded from the FSL_sampler() and input to the network remains the same 10 classes?

**Comment by priceee** [comment]
In the code you only used split 1 of the data, when you testing your model, are you testing on split 1~5 or just split 1?

**Comment by charusharma1991** [comment]
Thanks for your interest!
1. We use random classes in every run and we consider 10 such different splits to get the avg results mentioned in the paper.
2. We make different train-test splits. so, for one training set, we consider one test set.

**Comment by priceee** [comment]
> Thanks for your interest!
> 
> 1. We use random classes in every run and we consider 10 such different splits to get the avg results mentioned in the paper.
> 2. We make different train-test splits. so, for one training set, we consider one test set.

Thank you for your answer! I have another question in the segmentation tasks, in shapenet dataset there are 16 classes, however when running with 10-way-10-shot FSL learning, you can only randomly pick 10 classes. So how do you get the mIoU for each of the 16 classes when you can only randomly pick 10 classes every time?
